src/runtime/RegExpObject.cpp

```diff
@@ -532,4 +532,32 @@ void RegExpObject::pushBackToRegExpMatchedArray(ExecutionState& state, ArrayObje
         }
     }
 }
+
+RegExpStringIteratorObject::RegExpStringIteratorObject(ExecutionState& state, bool global, bool unicode, RegExpObject* regexp, String* string)
+    : IteratorObject(state, state.context()->globalObject()->regexpStringIteratorPrototype())
+    , is_global(global)
+    , is_unicode(unicode)
+    , is_done(false)
+    , m_regexp(regexp)
+    , m_string(string)
+{
+}
+
+void* RegExpStringIteratorObject::operator new(size_t size)
+{
+    static bool typeInited = false;
+    static GC_descr descr;
+    if (!typeInited) {
+        GC_word obj_bitmap[GC_BITMAP_SIZE(RegExpStringIteratorObject)] = { 0 };
```

Comment:
Could you call Object::fillGCDescriptor here?
instead of set a bit for object members

Reply:
`m_regexp`, `m_string` members need to be marked here too.